tests/Integration/Behaviour/Features/Scenario/Title/title_management.feature

```diff
@@ -0,0 +1,60 @@
+#./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s title
+@restore-all-tables-before-feature
+Feature: Title management
+  As an employee
+  I must be able to add, edit and delete titles
+
+  Scenario: Adding new title
+    When I define an uncreated title "titleNotFound"
+    When I add new title "3rdTitle" with following properties:
+      | name    | Third Title    |
+      | type    | Other          |
+#    Then title "thirdTitle" name should be "Normandy"
+#    Then title "thirdTitle" country should be "United titles"
+#    Then title "thirdTitle" zone should be "Europe"
+#    And title "thirdTitle" should be enabled
+#
+#  Scenario: Editing title
+#    When I edit title "thirdTitle" with following properties:
+#      | name    | Britain       |
+#      | enabled | false         |
+#      | country | Italy         |
+#      | zone    | South America |
+#    Then title "thirdTitle" name should be "Britain"
+#    Then title "thirdTitle" country should be "Italy"
+#    Then title "thirdTitle" zone should be "South America"
+#    And title "thirdTitle" should be disabled
```

Comment:
Why this part is commented?

Reply:
It will do in the next PR after this one.